setup.py

```diff
@@ -15,32 +15,27 @@
 setup(
     name="py42",
     version=about["__version__"],
-    description="The Official Code42 Python API Client",
-    long_description=readme,
-    long_description_content_type="text/markdown",
-    packages=find_packages("src"),
-    package_dir={"": "src"},
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4",
-    install_requires=["requests>=2.3"],
-    license="MIT",
     url="https://github.com/code42/py42",
     project_urls={
         "Issue Tracker": "https://github.com/code42/py42/issues",
         "Documentation": "https://py42docs.code42.com/",
         "Source Code": "https://github.com/code42/py42",
     },
+    description="The Official Code42 Python API Client",
```

Comment:
The word "official" may be superfluous. I believe it was removed from
the py42docs site too.